README.md

```diff
@@ -4,11 +4,26 @@
 
 _Run a Python API in this ready-to-use-repository in minutes_
 
-With this template respository you can quickly get hands-on a Python project for running an HTTP API using the [Fastapi](https://fastapi.tiangolo.com/) framework. It will let you focus in working with the project instead of setting and configuring anything.
```

Comment:
The premise of Codespaces is that you can get hands-on without configuring anything like say, a Python installation or dependencies. Can we keep this first line, or if you could suggest an improvement that doesn't remove the compelling reason of using Codespaces?

Reply:
Sure, added back, with a more explicit Codespaces reference, since I think the second sentence is more about Codespaces than FastAPI (right?)